onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -313,6 +314,8 @@ def quantize_static(
             List of nodes names to exclude. The nodes in this list will be excluded from quantization
             when it is not None.
         use_external_data_format: option used for large size (>2GB) model. Set to False by default.
+        convert_attribute (bool): If true, convert all tensors to external data
+                If false, convert only non-attribute tensors to external data
```

Comment:
```suggestion
        save_attributes (bool): When `use_external_data_format` is True, attribute (constant) tensors are included in the external data when this flag is True. Otherwise only non-attribute tensors are saved into the external data
```